inc/class-cli-command.php

```diff
@@ -74,11 +74,49 @@ public function delete( $args, $assoc_args ) {
 				$session->destroy();
 				WP_CLI::log( sprintf( 'Session destroyed: %s', $session_id ) );
 			} else {
-				WP_CLI::warning( sprintf( "Session doesn't exist: %s", $session_id ) );
+				WP_CLI::warning( sprintf( "Session doesn't exist: %s",
+					$session_id ) );
 			}
 		}
 	}
 
+	/**
+	 * Set id as primary key in the Native PHP Sessions plugin table.
+	 *
+	 * [<session-id>...]
+	 * : One or more session IDs
+	 *
+	 * [--all]
+	 * : Delete all sessions.
+	 *
+	 * @subcommand add-index
+	 */
+
+	public function add_index( $args, $assoc_arc ) {
+		global $wpdb;
+
+		if ( ! PANTHEON_SESSIONS_ENABLED ) {
+			WP_CLI::error( 'Pantheon Sessions is currently disabled.' );
```

Comment:
This should be internationalized.

```suggestion
			WP_CLI::error( __( 'Pantheon Sessions is currently disabled.', 'wp-native-php-sessions' );
```